runtime/src/main/cpp/ObjCExportErrors.mm

```diff
@@ -46,7 +46,7 @@ static void printlnMessage(const char* message) {
 static char kotlinExceptionOriginChar;
 
 static bool isExceptionOfType(KRef exception, const TypeInfo** types) {
-  for (int i = 0; types[i] != nullptr; ++i) {
+  if (types) for (int i = 0; types[i] != nullptr; ++i) {
```

Comment:
What is this patch for?
IIRC, `types` is guaranteed to be non-null.

Reply:
`types` comes from another module and has never been checked, so it is _supposed_ to be not null, not _guaranteed_. Using `gsl::not_null` would be better solution indeed (let's consider).  The patch arose as a result of my attempt (abandoned now) to implement "optional" behavior (similar to `std::optional`).